kvs-indexing/src/main/java/org/gbif/kvs/indexing/species/NameUsageMatchIndexer.java

```diff
@@ -105,34 +107,47 @@ private static void run(NameUsageMatchIndexingOptions options) {
                       new SaltedKeyGenerator(
                           storeConfiguration.getHBaseKVStoreConfiguration().getNumOfKeyBuckets());
 
-                  private transient NameMatchService nameMatchService;
+                  private transient ChecklistbankService checklistbankService;
 
                   private transient BiFunction<byte[], NameUsageMatch, Put> valueMutator;
 
                   @Setup
                   public void start() {
-                    nameMatchService = new NameMatchServiceSyncClient(nameMatchClientConfiguration);
+                    checklistbankService = new ChecklistbankServiceSyncClient(nameMatchClientConfiguration);
                     valueMutator =
                         NameUsageMatchKVStoreFactory.valueMutator(
                             Bytes.toBytes(storeConfiguration.getHBaseKVStoreConfiguration().getColumnFamily()),
                             Bytes.toBytes(storeConfiguration.getValueColumnQualifier()));
                   }
 
+                  /**
+                   * Retrieves the IUCN Red List category from the name usage match response.
+                   */
+                  private ThreatStatus getIucnRedListCategory(NameUsageMatch nameUsageMatch) {
+                    Integer usageKey = Objects.nonNull(nameUsageMatch.getAcceptedUsage()) ? nameUsageMatch.getAcceptedUsage().getKey() : nameUsageMatch.getUsage().getKey();
```

Comment:
I'm not 100% sure but I think there could be cases where nameUsageMatch is not null but both the acceptedUsage and the usage are null. Maybe it's worth to handle that case

Reply:
ok I will double check those nulls then

Reply:
https://github.com/gbif/key-value-store/commit/abb666a9a6b28f0fb8209f9f62115398c4188d21